navigation.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+import { View, Text } from 'react-native';
+import { StackNavigator, TabNavigator } from 'react-navigation';
+
+import App from './src/Screens/App';
+import Books from './src/Screens/Books';
+
+const navigationRoutes = {
+  TodoList: () => ({title: 'Todo List'}),
+  Books: () => ({title: 'Books'})
+}
+
+const Routes = TabNavigator(
+  {
+  Home: { screen: App, navigationOptions: navigationRoutes.TodoList},
+  Settings: { screen: Books, navigationOptions: navigationRoutes.Books},
+ },
+ {
+  tabBarPosition: 'bottom',
+  tabBarOptions: {
```

Comment:
I'd extract these options in another file so we do not merge components with styling